javascript/apps/taiga/src/app/modules/project/feature-kanban/components/workflow/kanban-workflow.component.css

```diff
@@ -39,3 +45,11 @@ tui-scrollbar {
 tg-kanban-story {
   margin: 0;
 }
+
+tg-ui-drag-in-progress tg-kanban-status {
+  block-size: var(--drag-status-height);
+  box-shadow: 3px 4px 14px 0 rgba(0, 138, 168, 0.15);
+  cursor: grab;
```

Comment:
I can't see the grab cursor
![Peek 2023-07-24 06-51](https://github.com/taigaio/taiga/assets/5970959/876cdc87-0696-4508-9815-dd13cd2d4c53)

Reply:
Also, when i'm dragging the card, the status column should have 'border' as it's 'focused'
![image](https://github.com/taigaio/taiga/assets/5970959/957a8800-e9cc-468e-99f0-ce3556cdd286)